src/pages/ScreeningsPage.vue

```diff
@@ -1,11 +1,156 @@
 <template>
-  <div></div>
+  <div>
+    <h1>Screenings</h1>
+    <h2>{{ selectedDay }}</h2>
+    <div>
+      <button
+        class="day-tabs"
+        v-for="day in daysList"
+        :key="day.id"
+        :data-date="day.date.toJSON()"
+        @click="setSelectedDay"
+      >
+        {{ day.id === 1 ? day.name : niceDay(day.name) }}
+      </button>
+      <button class="day-tabs__calendar">
+        <img src="@/assets/img/calendaricon.svg" alt="calendar-icon" />
+      </button>
+      <hr />
+      <div v-for="movieId in moviesOnTheDay" :key="`movie_${movieId}`">
+        <p>
+          {{
+            $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+              .title
+          }}
+        </p>
+        <p
+          v-for="seance in seancesList
+            .filter(
+              (seance) => seance.datetime.substring(0, 10) === selectedDay
+            )
+            .filter((seance) => seance.movie == movieId)"
```

Comment:
I would definitely create a method for this to not have much logic in the template